src/obtainAuthHeaders.ts

```diff
@@ -32,7 +32,7 @@ export async function getAuthFetcher(
   allowUnauthenticated = true
 ): Promise<AuthFetcher | { fetch: any }> {
   if (!oidcProviderCookie.length && allowUnauthenticated) {
-    return { fetch };
+    return { fetch, allowUnauthenticated };
```

Comment:
i would prefer `fetcher.authorized === false`, so that
* don't use negatives in variable names
* the 'allow' is dropped because we don't want to know whether it's allowed, but whether it's true

I will commit a fix to this branch

Reply:
yes, sounds good to me.